src/main/java/Messenger/MessengerNotification.java

```diff
@@ -0,0 +1,57 @@
+package Messenger;
+
+import Notification.Notification;
+
+import java.util.Date;
+
+
+/**
```

Comment:
Will we be keeping these comments? I mean I would opt for removing these from intelliJ template settings, so the wouldn't be generated since this information is in source control and comments are ugly in general.